{
  target: {
    RequestScopeIdentifier: {
      ScopeId: Guid_1
    },
    EventHandlers: [
      {
        ServiceScopeId: Guid_2,
        WorkerStatus: Faulted,
        IsDisposed: true
      },
      {
        ServiceScopeId: Guid_2,
        WorkerStatus: Faulted,
        IsDisposed: true
      },
      {
        ServiceScopeId: Guid_2,
        WorkerStatus: Done,
        IsDisposed: true
      },
      {
        ServiceScopeId: Guid_2,
        WorkerStatus: Done,
        IsDisposed: true
      }
    ],
    MockTaskRunner: [
      {
        Method: TaskRunner.WhenAll(IEnumerable<Func<Task>> tasks),
        Arguments: [
          [
            {
              Type: Func<Task>,
              Target: EventBus.<>c__DisplayClass6_1,
              Method: System.Threading.Tasks.Task Publish()
            },
            {
              Type: Func<Task>,
              Target: EventBus.<>c__DisplayClass6_1,
              Method: System.Threading.Tasks.Task Publish()
            },
            {
              Type: Func<Task>,
              Target: EventBus.<>c__DisplayClass6_1,
              Method: System.Threading.Tasks.Task Publish()
            },
            {
              Type: Func<Task>,
              Target: EventBus.<>c__DisplayClass6_1,
              Method: System.Threading.Tasks.Task Publish()
            }
          ]
        ],
        ReturnValue: {
          Status: RanToCompletion
        }
      }
    ]
  },
  logs: [
    {
      Error: An exception was thrown while executing Reprise.UnitTests.Features.Events.EventBusTests.MockEventHandler.,
      Category: EventBus,
      State: [
        {
          EventHandlerType: Reprise.UnitTests.Features.Events.EventBusTests.MockEventHandler
        },
        {
          {OriginalFormat}: An exception was thrown while executing {EventHandlerType}.
        }
      ],
      Exception: {
        Type: Exception,
        Message: Test message
      }
    },
    {
      Error: An exception was thrown while executing Reprise.UnitTests.Features.Events.EventBusTests.MockEventHandler.,
      Category: EventBus,
      State: [
        {
          EventHandlerType: Reprise.UnitTests.Features.Events.EventBusTests.MockEventHandler
        },
        {
          {OriginalFormat}: An exception was thrown while executing {EventHandlerType}.
        }
      ],
      Exception: {
        Type: Exception,
        Message: Test message
      }
    }
  ]
}